Screenbox/Pages/ArtistDetailsPage.xaml

```diff
@@ -219,7 +229,8 @@
             Grid.Row="1"
             Padding="{StaticResource PageContentMargin}"
             extensions:ListViewExtensions.ItemCornerRadius="8"
-            extensions:ListViewExtensions.ItemMargin="0,2,0,2"
+            extensions:ListViewExtensions.ItemMargin="0,0,0,4"
```

Comment:
This change broke vertical alignment for items on Windows 11

![image](https://github.com/huynhsontung/Screenbox/assets/31434093/84a3496e-ebb3-4b72-a88f-cdf658c34e53)

Reply:
Yeah, now I'm curious about what would happen if I add it to the #288 style. Reverted for now.